samples/stand-alone-samples/shadow/shadow_sync/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+cmake_minimum_required(VERSION 3.1)
+project(shadow-sync CXX)
+
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_INSTALL_PREFIX}/lib/cmake")
+
+file(GLOB SHADOW_SYNC_SRC
+       "*.cpp"
+)
+
+add_executable(${PROJECT_NAME} ${SHADOW_SYNC_SRC})
+
+set_target_properties(${PROJECT_NAME} PROPERTIES
+    LINKER_LANGUAGE CXX
+    CXX_STANDARD 11)
```

Comment:
opinion: we should start customers with 14 or 17

Reply:
C++17 please. I've tried compiling very basic examples under 17 and they fail with AWS_ERROR_SUCCESS. If I rebuild the sdk changing 11 to 17, my code works fine. Is there a way to simply override CXX_STANDARD in the lib? I don't want to fork the sdk just to get C++17, but I need it for my project.

Reply:
some of CI does not support C++17... So, probably you can make the change locally? Here is the [followup PR](https://github.com/aws/aws-iot-device-sdk-cpp-v2/pull/177)